src/strings/PowerFxResources.en-US.resx

```diff
@@ -4448,4 +4448,16 @@
   <data name="AboutUniChar_column_of_numbers" xml:space="preserve">
     <value>A table column of numeric values to be converted into Unicode characters.</value>
   </data>
+  <data name="ErrOnlyPartialAttribute" xml:space="preserve">
+    <value>Unsupported attribute kind</value>
+    <comment>Error message when the user attempted to use the attribute syntax on a named formula definition, but we didn't recognize the attribute name they specified.</comment>
+  </data>
+  <data name="ErrOperationDoesntMatch" xml:space="preserve">
+    <value>All Partial operations on matching named formulas must be the same.</value>
+    <comment>{Locked=Partial} Error message when the user attempted to use a partial operator on multiple named formulas, but didn't specify the same operator on each one.</comment>
+  </data>
+  <data name="ErrUnknownPartialOp" xml:space="preserve">
+    <value>Unknown Partial operation.</value>
+    <comment>{Locked=Partial}. Error message when the user attempted to use a Partial operator that we don't recognize.</comment>
```

Comment:
>Error message when the user attempted to use a Partial operator that we don't recognize [](http://example.com/codeflow?start=31&length=87)

Please give more context about what this is - translators may not know what a "Partial" operat[ion/or] is and may translate it incorrectly.